### ISSUE TITLE
Access Denied Nextjs : No solution seems to work

### ISSUE THREAD

**Issue by BahauddinAziz** [aws-amplify/amplify-console#2415]
### Before opening, please confirm:

- [X] I have checked to see if my question is addressed in the [FAQ](https://github.com/aws-amplify/amplify-console/blob/master/FAQ.md).
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-console/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-console/blob/master/CONTRIBUTING.md).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### Amplify Console feature

Not applicable

### Describe the bug

I get the following error

AccessDenied: Access Denied
                                  at Request.extractError (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/services/s3.js:699:35)
                                  at Request.callListeners (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
                                  at Request.emit (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
                                  at Request.emit (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/request.js:686:14)
                                  at Request.transition (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/request.js:22:10)
                                  at AcceptorStateMachine.runTo (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/state_machine.js:14:12)
                                  at /root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/state_machine.js:26:10
                                  at Request.<anonymous> (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/request.js:38:9)
                                  at Request.<anonymous> (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/request.js:688:12)
                                  at Request.callListeners (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/sequential_executor.js:116:18)
                                  at Request.emit (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
                                  at Request.emit (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/request.js:686:14)
                                  at Request.transition (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/request.js:22:10)
                                  at AcceptorStateMachine.runTo (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/state_machine.js:14:12)
                                  at /root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/state_machine.js:26:10
                                  at Request.<anonymous> (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/request.js:38:9)
                                  at Request.<anonymous> (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/request.js:688:12)
                                  at Request.callListeners (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/sequential_executor.js:116:18)
                                  at callNextListener (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/sequential_executor.js:96:12)
                                  at IncomingMessage.onEnd (/root/.//node_modules/@sls-next/aws-s3/node_modules/aws-sdk/lib/event_listeners.js:335:13)
                                  at IncomingMessage.emit (events.js:326:22)
                                  at IncomingMessage.EventEmitter.emit (domain.js:483:12)
                                  at endReadableNT (_stream_readable.js:1241:12)
                                  at processTicksAndRejections (internal/process/task_queues.js:84:21) {
                                  code: 'AccessDenied',
                                  region: null,
                                  time: 2021-11-29T07:36:37.023Z,
                                  requestId: 'FP66RQBNXGJJK7K0',
                                  extendedRequestId: 'Hwt6vz2mu8pc3pyipBngfXRbd/eeVyw8XYUZWPNhF4fkX4f+2aSBP8XvwryEWNQ+vsxz+oITfw4=',
                                  cfId: undefined,
                                  statusCode: 403,
                                  retryable: false,
                                  retryDelay: 22.001353601658803
                                  }

### Expected behavior

The app should build

### Reproduction steps

Deploy any sample nextjs app through aws amplify console.

### Build Settings

```yaml
version: 1
frontend:
  phases:
    preBuild:
      commands:
        - npm ci
    build:
      commands:
        - npm run build
  artifacts:
    baseDirectory: .next
    files:
      - '**/*'
  cache:
    paths:
      - node_modules/**/*
```

### Additional information

I have tried all the solutions including:

1. extending permissions to admin
2. Adding inline policies
3. Adding individual policies
4. Nextjs version check

**Comment by aleksey-shmatov** [comment]
I had this issue three times and all times for different reasons.
Initially it was because of permissions issues, but seems like you already tried that.
Second reason was that for some unknown reason my root account did not have permissions to enable "enable transfer acceleration" on the deployment bucket. I have contacted support and they gave permissions to my account to enable this feature.
Third reason happened after some limitations were set on my account because of suspicious activity. Most of the limitations were lifted but somehow I still did not have access to cloud front distribution.

**Comment by hloriana** [comment]
Hi @BahauddinAziz can you provide your AppID and the region you're trying to deploy your app in?

**Comment by sananbintahir** [comment]
I've been having the same issue for the past day. Just started happening out of nowhere.

AppID: dwfmevspagm4e
Region: eu-central-1

**Comment by hloriana** [comment]
@sananbintahir can you make sure the IAM role associated with your app has the right permissions? Please see reference: [[ERROR] AccessDenied: Access Denied](https://github.com/aws-amplify/amplify-console/blob/main/FAQ.md#error-accessdenied-access-denied)

**Comment by sananbintahir** [comment]
> @sananbintahir can you make sure the IAM role associated with your app has the right permissions? Please see reference: [[ERROR] AccessDenied: Access Denied](https://github.com/aws-amplify/amplify-console/blob/main/FAQ.md#error-accessdenied-access-denied)

Yes, the IAM role has admin permissions, so that’s not the problem. The issue is only happening with newly connected branches, the ones that were already connected from more than a day ago are building fine

**Comment by hloriana** [comment]
Hi @sananbintahir, I was able to successfully build my SSR app on new branches. Can you provide reproduction steps so we can understand this behavior?

**Comment by sananbintahir** [comment]
@hloriana I had to contact aws support yesterday to resolve the issue. According to support, “The reason behind that behavior was a trust level issue on your AWS account.”

After support increased the trust level on my account, my builds started working again. Still not sure what caused this trust issue, will follow it up with support to figure it out. Weirdly enough, every AWS service, besides the SSR deployment on amplify was working fine for me.

**Comment by hloriana** [comment]
Hi @sananbintahir okay glad to hear AWS Support was able to unblock you. I'm going to close this issue but if you run into anything else or have a question feel free to open a new issue. Thanks!
